commit ba43de3f9b1ba7cad07bef08915b1c26693de4bf
Author: Jelmer VernooÄ³ <jelmer@jelmer.uk>
Date:   Thu Feb 25 04:22:55 2016 +0000

    Add make variable for r.js, so it can easily be overriden.
    
    This is useful for platforms like Debian that don't ship with r.js in
    the path.

diff --git a/Makefile b/Makefile
index 9cb3175..1f9dc50 100644
--- a/Makefile
+++ b/Makefile
@@ -25,6 +25,8 @@ DOCS_MAN_DST := man/man1/isso.1 man/man5/isso.conf.5
 
 DOCS_HTML_DST := docs/_build/html
 
+RJS = r.js
+
 all: man js site
 
 init:
@@ -37,10 +39,10 @@ check:
 	-@python3 -m pyflakes $(ISSO_PY_SRC)
 
 isso/js/%.min.js: $(ISSO_JS_SRC) $(ISSO_CSS)
-	r.js -o isso/js/build.$*.js out=$@
+	$(RJS) -o isso/js/build.$*.js out=$@
 
 isso/js/%.dev.js: $(ISSO_JS_SRC) $(ISSO_CSS)
-	r.js -o isso/js/build.$*.js optimize="none" out=$@
+	$(RJS) -o isso/js/build.$*.js optimize="none" out=$@
 
 js: $(ISSO_JS_DST)
 
